test/lib/mayaUsd/render/mayaToHydra/testMayaLights.py

```diff
@@ -26,7 +26,14 @@ class TestMayaLights(mtohUtils.MayaHydraBaseTestCase): #Subclassing mtohUtils.Ma
     _file = __file__
 
     IMAGE_DIFF_FAIL_THRESHOLD = 0.01
-    IMAGE_DIFF_FAIL_PERCENT = 0.2
+
+    @property
+    def IMAGE_DIFF_FAIL_PERCENT(self):
+        # HYDRA-943 : Shadows don't seem to work properly on OSX, possibly only with point lights.
+        # Only the "all lights with shadows" check fails, and shadows with point light only is skipped.
+        if platform.system() == "Darwin":
+            return 3
+        return 0.2
```

Comment:
Unfortunately had to bring back the workaround in this case, as the higher threshold we used hid another issue. This has likely been happening for a while, unsure if this is only a test issue or if it is also an issue when using MayaHydra normally.